.htaccess

```diff
@@ -1,240 +1,561 @@
 ############################################
-## GoDaddy specific options
+## Default index file
+
+	DirectoryIndex index.php
+
+############################################
+## Custom error pages
 
-#   Options -MultiViews
+	#ErrorDocument 401 /errors/401.php
+	#ErrorDocument 403 /errors/403.php
+	#ErrorDocument 404 /errors/404.php
+	#ErrorDocument 405 /errors/405.php
+	#ErrorDocument 410 /errors/410.php
+	#ErrorDocument 500 /errors/500.php
+	#ErrorDocument 503 /errors/503.php
 
-## you might also need to add this line to php.ini
-##     cgi.fix_pathinfo = 1
-## if it still doesn't work, rename php.ini to php7.ini
+############################################
+## Prevent character encoding issues from server overrides
+## If you still have issues use the second line instead
+
+	AddDefaultCharset Off
+	#AddDefaultCharset UTF-8
 
 ############################################
-## default index file
+## Uncomment if running in cluster environment
+## http://developer.yahoo.com/performance/rules.html#etags
 
-    DirectoryIndex index.php
+	#FileETag none
 
 ############################################
-## php7 settings
+## GoDaddy specific options
+## You might also need to add this line to php.ini
+##    cgi.fix_pathinfo = 1
+
+	#Options -MultiViews
 
 <IfModule mod_php7.c>
 
 ############################################
-## adjust max execution time
+## PHP7 Settings
+
+############################################
+## Adjust max execution time
 
-    php_value max_execution_time 18000
+	#php_value max_execution_time 18000
 
 ############################################
-## disable automatic session start
-## before autoload was initialized
+## Disable automatic session start before autoload was initialized
 
-    php_flag session.auto_start off
+	php_flag session.auto_start off
 
 ############################################
-## enable resulting html compression
+## Enable resulting html compression
 
-    #php_flag zlib.output_compression on
+	#php_flag zlib.output_compression on
 
 ###########################################
-# disable user agent verification to not break multiple image upload
+## Disable user agent verification to not break multiple image upload
 
-    php_flag suhosin.session.cryptua off
+	php_flag suhosin.session.cryptua off
+
+</IfModule>
+
+<IfModule mod_php8.c>
```

Comment:
Is this actually correct? It seems that it might just be `<IfModule php_module>` for PHP8.

https://php.watch/versions/8.0/mod_php-rename

Reply:
Do you have php8.x installed? If so, please check the name of the php module used by Apache. I didn't switch to php8 yet. If it's like that guy says we have to change not only here but also in the htaccess files I worked on recently. It is not critical but it must be resolved.

Reply:
I haven't migrated yet, but also am not using mod_php but instead php-fpm. And yes, I did approve the other PR for 20.0 in the js directory figuring that was the correct IfModule, but maybe it's not.

Reply:
I've tested today, for php7 we've to use "mod_php7.c", for php8 we've to use "mod_php.c", any other won't work.